packages/wonder-blocks-dropdown/__tests__/generated-snapshot.test.js

```diff
@@ -669,7 +669,7 @@ describe("wonder-blocks-dropdown", () => {
             darkBackgroundWrapper: {
                 flexDirection: "row",
                 justifyContent: "flex-end",
-                backgroundColor: Color.darkBlue,
+                background: Color.darkBlue,
```

Comment:
Why the switch to `background`?

Reply:
For some reason, the tests were throwing a warning about using `background` and `backgroundColor`, but I don't remember what was, and now I don't see it, so I 'll revert back to `bgColor`.